packages/datatrak-web-server/src/routes/TaskRoute.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate';
+import { DatatrakWebTaskRequest } from '@tupaia/types';
+import { TaskT, formatTaskResponse } from '../utils';
```

Comment:
I think that it could be nice to co-locate all of the Task related route code in a `routes/tasks` folder since the formatTaskResponse and formatTaskChanges code relates exclusively to those endpoints. I don't feel too strongly about it though so I don't mind if you would rather leave it as is.

Reply:
@alexd-bes any thoughts on this one?

Reply:
Oh yes, sorry. I thought about putting them in a folder at the time, but I wasn't sure because all the tasks endpoints are different, and we don't have, for instance a folder for `projects` with the `ProjectsRoute` and the `ProjectRoute` in it ( or any other cases either, except survey submission) so I decided against it